### PR TITLE
fix(menu): fix missing menu

### LINKED ISSUE #260: No dedicated menu on "Authentication for remote devices (VMware)" page
### Describe the bug

The GLPI Inventory plugin menu is not present when managing "Authentication for remote devices (VMware)".

### To reproduce

1. Click on "Network" -> "Authentication for remote devices (VMware)" menu
image
2. See the GLPI Inventory plugin menu is not rendered
image

### Expected behavior

GLPI Inventory menu is still present.

### Operating system

Linux

### GLPI Agent version

1.4

### GLPI version

10.0.5

### GLPIInventory plugin

1.0.5

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by stonebuzz**
Fix missing menu

image

from 

- ```PluginGlpiinventoryCredential```
- ```PluginGlpiinventoryCredentialIp```

fix #260

### CODE REVIEW

**Review by orthagh — APPROVED** [review]

**Comment by g-bougard** [comment]
The issue remains when editing a credential: `/plugins/glpiinventory/front/credential.form.php?id=1`

**Review by cedric-anne — APPROVED** [review]

**Review by g-bougard — CHANGES_REQUESTED** [review]
Menus are still missing when editing a credential.

**Comment by stonebuzz** [comment]
@g-bougard 
can you retry ?

**Comment by g-bougard** [comment]
That's okay now.

**Review by cedric-anne — CHANGES_REQUESTED** [review]
Instead of copy/pasting the whole `dropdown.common.form.php` code, maybe you could just do
```php
if (!empty($_POST) || isset($_GET['_in_modal'])) {
    include(GLPI_ROOT . "/front/dropdown.common.form.php");    
} else {
    Html::header(
        __('GLPI Inventory', 'glpiinventory'),
        $_SERVER["PHP_SELF"],
        "admin",
        "pluginglpiinventorymenu",
        "credential"
    );
    PluginGlpiinventoryMenu::displayMenu("mini");
    //If there is no form to submit, display the form
    $dropdown->display($_GET);
    Html::footer();
}

**Review by g-bougard — APPROVED** [review]

**Comment by stonebuzz** [comment]
it is because of the 

> Instead of copy/pasting the whole `dropdown.common.form.php` code, maybe you could just do
> 
> ```
> if (!empty($_POST) || isset($_GET['_in_modal'])) {
>     include(GLPI_ROOT . "/front/dropdown.common.form.php");    
> } else {
>     Html::header(
>         __('GLPI Inventory', 'glpiinventory'),
>         $_SERVER["PHP_SELF"],
>         "admin",
>         "pluginglpiinventorymenu",
>         "credential"
>     );
>     PluginGlpiinventoryMenu::displayMenu("mini");
>     //If there is no form to submit, display the form
>     $dropdown->display($_GET);
>     Html::footer();
> }
> ```

indeed

File: front/credential.form.php [hunk 1]
```diff
@@ -31,11 +31,21 @@
...
+        $_SERVER["PHP_SELF"],
+        "admin",
+        "pluginglpiinventorymenu",
+        "credential"
+    );
+    PluginGlpiinventoryMenu::displayMenu("mini");
+    //If there is no form to submit, display the form
+    $dropdown->display($_GET);
+    Html::footer();
 }
```
Comment (cedric-anne) [inline front/credential.form.php:51]: Not sure it is used, but it was here previously.
```suggestion
}

if (isset($_GET['popup'])) {
    Ajax::refreshDropdownPopupInMainWindow();
}
```

File: front/credentialip.form.php [hunk 2]
```diff
@@ -31,10 +31,21 @@
...
  */
 
-include("../../../inc/includes.php");
+include('../../../inc/includes.php');
+$dropdown = new PluginGlpiinventoryCredentialIP();
 
-$dropdown = new PluginGlpiinventoryCredentialIp();
-if (empty($_POST['plugin_glpiinventory_credentials_id'])) {
-    $_POST['plugin_glpiinventory_credentials_id'] = 0;
+if (!empty($_POST) || isset($_GET['_in_modal'])) {
```
Comment (cedric-anne) [inline front/credentialip.form.php:37]:
```suggestion
Same here.
if (empty($_POST['plugin_glpiinventory_credentials_id'])) {
    $_POST['plugin_glpiinventory_credentials_id'] = 0;
}

if (!empty($_POST) || isset($_GET['_in_modal'])) {
```

File: front/credential.form.php (hunk 1 above)
Comment (stonebuzz) [reply front/credential.form.php:51]: done

File: front/credentialip.form.php (hunk 2 above)
Comment (stonebuzz) [reply front/credentialip.form.php:37]: done

**Review by cedric-anne — APPROVED** [review]

**Review by g-bougard — APPROVED** [review]

**Pull request merged**